lib/ResponseBuilder.js

```diff
@@ -179,8 +194,8 @@ class ResponseBuilder {
     };
 
     try {
-      const url = await BPromise
-        .fromCallback(cb => this.s3client.getSignedUrl('getObject', getParams, cb));
+      const url = this.s3client.getSignedUrl('getObject', getParams);
```

Comment:
callback was not asynchronous, supports sync call anyway

Reply:
From the docs:
> Note: You must ensure that you have static or previously resolved credentials if you call this method synchronously (with no callback), otherwise it may not properly sign the request. If you cannot guarantee this (you are using an asynchronous credential provider, i.e., EC2 IAM roles), you should always call this method with an asynchronous callback.

Reply:
Thus, I'm a bit concerned to use synchronous version

Reply:
no prob I change it back